src/main/java/frc/robot/commands/DisposeCommand.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Acquisition;
+
+public class DisposeCommand extends Command {
+    private Acquisition acquisition = Acquisition.getInstance();
+
+    public DisposeCommand() {
+        super.addRequirements(acquisition);
+    }
+
+    @Override
+    public void execute() {
+        acquisition.dispose();
+    }
+
+    @Override
+    public boolean isFinished() {
+        return !(acquisition.getTopLaser() && acquisition.getBottomLaser());
```

Comment:
```suggestion
        return !acquisition.getTopLaser() && !acquisition.getBottomLaser();
```
This is a bit more readable